packages/react-components/react-tabs/src/components/Tab/useTabStyles.ts

```diff
@@ -16,6 +16,13 @@ const reservedSpaceClassNames = {
   content: 'fui-Tab__content--reserved-space',
 };
 
+// These should match the constants defined in @fluentui/react-icons
+// This package avoids taking a dependency on the icons package for only the constants.
+const iconClassNames = {
```

Comment:
I think react-tabs should avoid a dependency on react-icons at least for now so I'm willing to manage declaring the constant here.  Filed this issue to track fixing it: https://github.com/microsoft/fluentui/issues/25598